evm/src/main/java/org/hyperledger/besu/evm/MainnetEVMs.java

```diff
@@ -456,5 +459,8 @@ public static void registerShandongOperations(
     registerParisOperations(registry, gasCalculator, chainID);
     // Register the PUSH0 operation.
     registry.put(new Push0Operation(gasCalculator));
+    registry.put(new RelativeJumpOperation(gasCalculator));
```

Comment:
I agree it will be better for the history, but as it's not clear what would end up in each network (i.e. withdrawals, eof, shangai-testnet) maybe we can keep this under Shandong and then split out this one into the new ones (and finally rename it to whatever the eof network will be)